pkg/types/baremetal/validation/platform.go

```diff
@@ -204,6 +204,14 @@ func validateHostsBase(hosts []*baremetal.Host, fldPath *field.Path, filter vali
 		return !valueFound
 	})
 
+	validate.RegisterValidation("mac", func(fl validator.FieldLevel) bool {
```

Comment:
Isn't `mac` an already baked-in validation? `RegisterValidation` should be used just for custom validations AFAIK

Reply:
I'd expect the same thing but after adding https://github.com/openshift/installer/pull/5652/files#diff-534defd3e5e7453c872c0992bd665e8c6b241d7f3baab87d8893dc0995829cbfR39 and running this test https://github.com/openshift/installer/pull/5652/files#diff-534defd3e5e7453c872c0992bd665e8c6b241d7f3baab87d8893dc0995829cbfR39. It failed. I think it is not baked-in validation or there is something I could not see.

Reply:
Probably it was a typo or cached test result? AFAIK it's a baked-in validator, so I'd expect to be invoked automatically in the `validateHostsBase`. I'd suggest giving it a try by removing the registration

Reply:
As you said it is baked-in validation but it just needs this;

https://github.com/openshift/installer/pull/5652/files#diff-3294dd7b4b68c688149586052c12a8cf764e28ab6ada68f60138f10fc331c4e4R221-R222

Now I think it is ready for one more review.